gulp-test-chrome-debug.js

```diff
@@ -0,0 +1,75 @@
+const { series } = require('gulp');
+const del = require('del');
+const run = require('gulp-run-command').default;
+
+const IS_WINDOWS = (process.platform == 'win32');
+
+/**
+ * Run a command in the vscode-chrome-debug working directory
+ */
+function runInDebug(cmd) {
```

Comment:
I'd rename this to runInChromeDebug. My mind goes to a very different places with runInDebug...